drivers/misc/sky13317/sky13317.c

```diff
@@ -49,7 +49,7 @@ const RFCC26XX_HWAttrsV2 RFCC26XX_hwAttrs = {
 
 PINCTRL_DT_INST_DEFINE(0);
 DEVICE_DT_INST_DEFINE(0, board_antenna_init, NULL, NULL, NULL,
-					  POST_KERNEL, CONFIG_BOARD_ANTENNA_INIT_PRIO, NULL);
+					  POST_KERNEL, CONFIG_SKY13317_INIT_PRIO, NULL);
 
 static const struct pinctrl_dev_config *ant_pcfg = PINCTRL_DT_INST_DEV_CONFIG_GET(0);
```

Comment:
I have a problem with this driver, this is not a universal driver, no other platform can make use of this driver, the dts description lacks any sort of description for it and looking at the source file it essentially fits a device which has 2.4G output, PA output and sub-GHz output, so isn't at all generic and can't be used by any other board. @gmarull

Reply:
What should be added to the dts description? While the gpios is fixed (max 3), the pinctrl can be 0-8 depending on how the device is used.

I wonder if the driver can be changed to have nothing about the type of network it is configuring. Can you provide anything similar that can serve as an example. 
Alternatively, would it be fine to have three copies of the same `board_antenna` file?